storage/minio/egg-minio-s3.json

```diff
@@ -16,7 +16,7 @@
     "startup": ".\/minio.sh",
     "config": {
         "files": "{}",
-        "startup": "{\r\n    \"done\": \"guide\"\r\n}",
+        "startup": "{\r\n    \"done\": \"guide\",\r\n    \"done\": \"Documentation:\"\r\n}",
```

Comment:
Is having two "done" values valid? Does minio print one or other other depending on circumstances? If not, we should narrow it down to one that we're sure indicates that it started without error.

Reply:
Yeah, it allows multiple. I have an egg with lots of different possible startup values (I think just over a dozen now) it's not pretty but it DOES work. 

Tested this egg myself today as I couldn't handle perpetual startup. 

I don't want people updating their egg but not their version of minio and having the same issue by removing the first rendition of startup, given this change wasn't long ago on Minio's part.

Reply:
Awesome; thanks for the explanation. Backwards compatibility is always nice.